modules/openapi-generator/src/main/resources/kotlin-multiplatform-client/interface_opt_var.mustache

```diff
@@ -0,0 +1,6 @@
+{{#description}}
+    /**
+     * {{{description}}}
+     */
+{{/description}}
+    @SerialName(value = "{{{vendorExtensions.x-base-name-literal}}}") {{>modelMutable}} {{{name}}}: {{#isEnum}}{{#isListContainer}}{{#isList}}kotlin.collections.List{{/isList}}{{^isList}}kotlin.Array{{/isList}}<{{classname}}.{{{nameInCamelCase}}}>{{/isListContainer}}{{^isListContainer}}{{classname}}.{{{nameInCamelCase}}}{{/isListContainer}}{{/isEnum}}{{^isEnum}}{{{dataType}}}{{/isEnum}}?
```

Comment:
https://github.com/OpenAPITools/openapi-generator/pull/6916/commits/c8d7395706893b5fefd1d258043b7a7ae8618555 a bit improve property templates for ability to add @Contextual or @Serializable(with = Adapter::class) annotations

Reply:
I don't quite understand what you are getting at